lms/djangoapps/commerce/api/v0/views.py

```diff
@@ -150,13 +150,11 @@ def post(self, request, *args, **kwargs):
         # Make the API call
         try:
             # Pass along Sailthru campaign id
-            campaign_cookie = request.COOKIES.get(SAILTHRU_CAMPAIGN_COOKIE)
-            if campaign_cookie:
-                cookie = {SAILTHRU_CAMPAIGN_COOKIE: campaign_cookie}
-                if api_session.cookies:
-                    requests.utils.add_dict_to_cookiejar(api_session.cookies, cookie)
-                else:
-                    api_session.cookies = requests.utils.cookiejar_from_dict(cookie)
+            self._add_request_cookie_to_api_session(api_session, request, SAILTHRU_CAMPAIGN_COOKIE)
```

Comment:
do you know if there is a reason why this cookie does not have a configuration model?